plugin/trino-blackhole/src/test/java/io/trino/plugin/blackhole/TestBlackHoleSmoke.java

```diff
@@ -367,6 +369,22 @@ public void pageProcessingDelay()
         assertThatQueryDoesNotReturnValues("DROP TABLE nation");
     }
 
+    @Test
+    public void testMultipleSplits()
+    {
+        assertUpdate("CREATE TABLE table_multiple_splits (a integer) WITH (split_count = 5, pages_per_split = 3, rows_per_page = 2)");
```

Comment:
TestTable ?

Reply:
good point. here i am matching style of existing tests.